### ISSUE TITLE
Populate Admin dashboard list views without first requiring a search

### ISSUE THREAD

**Issue by jimmybrancaccio** [gitpod-io/gitpod#8983]
## Is your feature request related to a problem? Please describe

Showing nothing in the Admin Dashboard on the Workspaces page has resulted in some confusion that no workspaces exist on an instance or that there's a problem with the UI or a backend process that may fetch a list of workspaces, as one expects a list of workspaces to show up when the page is loaded. At the moment this page is only populated when a search is performed. This action which must be performed manually is not clearly identified on the page.

## Describe the behaviour you'd like

I'd like the page to display a list of workspaces perhaps 25 to 50 at a time in a paginated table.

## Describe alternatives you've considered

None as there is no other UI in the admin dashboard which displays a listing of workspaces.

## Additional context

N/A.

**Comment by jldec** [comment]
This issue is generalizable across lists E.g. 
- user list should default to you
- other lists should default to showing something sensible (most recent X?) as well

**Comment by jimmybrancaccio** [comment]
@jldec I totally agree. After putting together my other feature request for the users listing, I also saw the Projects and Teams pages so this idea would also apply to them as well.

The user list should default to something sensible as well, not just you. I'm thinking all users, sorted alphabetically, paginated. Ideally that goes for workspaces, teams and projects too.

**Comment by gtsiolis** [comment]
Usually, the most common approach would be listing with pagination most recently created entries for users, workspaces, projects, and teams. Depending on the entity (user, team, etc) we could also consider using a different default sorting filter like **recently created** or **recently updated**.

**Comment by jldec** [comment]
scheduled

**Comment by jimmybrancaccio** [comment]
Hey @svenefftinge! Are we going to reimplement this functionality in another issue?

**Comment by svenefftinge** [comment]
FWIW: I disabled the initial search because the search is super slow on gitpod.io and worth to be avoided when not needed.

**Comment by jimmybrancaccio** [comment]
So it looks like on the Users page we just show the users name, their email address and the time their Gitpod account was created. I don't remember how many accounts we pull for the first page but pulling 50 ran in 122ms for me (querying against the gitpod.io SaaS database):

```
SELECT name, d_b_identity.primaryEmail, creationDate FROM d_b_user INNER JOIN d_b_identity ON d_b_user.id =  d_b_identity.userId ORDER BY creationDate DESC LIMIT 50;
```

This could be a 🛹 to at least show the last 50 created accounts on the page without having to run a search first.
